src/bookshelf/rebar.lock

```diff
@@ -6,7 +6,7 @@
   0},
  {<<"chef_secrets">>,
   {git,"git://github.com/chef/chef_secrets.git",
-       {ref,"58373899212a23e4f37070aaa6d3a751b1281492"}},
+       {branch,"master"}},
   0},
```

Comment:
looks like ibrowse in ➜  chef-server git:(master) ✗ ack "ibrowse.git"
src/bookshelf/rebar.lock
42:  {git,"git://github.com/cmullaparthi/ibrowse.git",
points to the upstream, I think this can be changed to chef/ibrowse:chef-server branch once we figure out what brings this in.

Reply:
https://github.com/chef/mini_s3/commit/9899ce881a5056d18ea7ab6cfb252ecfbf50c8c2 - let a create an issue to check that this has indeed been resolved.

Reply:
Apparently mini_s3 uses cmullaparthi/ibrowse.
 Additional investigation revealed that the upcoming sigv4 work does not appear to reference ibrowse.
 PROPOSED RESOLUTION: Do nothing with mini_s3. Merge of sigv4 should resolve the issue.